assets/dev/scss/frontend/_swiper.scss

```diff
@@ -343,6 +343,11 @@ button.swiper-pagination-bullet {
 		height: 100%;
 	}
 
+	.elementor-swiper-button {
+		// Fix for Safari - Navigation arrows don't appear in cube mode without adding a 1px translateZ
+		transform: translate3d( 0, -50%, 1px );
```

Comment:
Chain it to an existing rule at line 510

Reply:
Why? This is only relevant for cube mode